### ISSUE TITLE
Backdrop is accessible by navigating using tabs though modal is open. 

### ISSUE THREAD

**Issue by lokeshreddy1706** [biig-io/ngx-smart-modal#192]
**I'm submitting an issue**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
Even though [backdrop] parameter is provided, when the modal popup is open,  we are able to access backdrop by clicking tabs. 

**Expected behavior**
Backdrop can't be accessible which clicking on tabs. The focus should move to the modal instead of the backdrop

**Reproduction of the problem**

1. Open any modal
2. click tabs multiple times until navigation goes to the backdrop

**<ngx-smart-modal [closable]="false" [backdrop]="true" identifier="sampleModal">**

**What is the motivation/use case for changing the behavior?**
To deliver the best quality product

**Please tell us about your environment:**
All

* **Smart Modal version:** 7.1.1

* **Angular version:** 7.3.3

* **Browser:** [all ]

* **Language:** [TypeScript ]

**Comment by maximelafarie** [comment]
Hi @lokeshreddy1706 and thank you for this well-detailed issue.

Indeed, it seems it's an accessibility bug and we should fix it asap. We need to make backdrop excluded from browser-readable elements to avoid it to be selected. 

I'm pushing this issue into our roadmap. I have good hope to fix that in our next incoming minor release.

Stay tuned!

Cheers! 🖖

**Comment by maximelafarie** [comment]
Closed by #238! 🎉🖖
